README.md

```diff
@@ -23,11 +23,37 @@ Usage
 -----
 
 ```
-$ ./check_url.py example.csv
+$ ./check_url.py ./list_url.csv
+
```

Comment:
Why with there empty lines?

Reply:
Because of 

```python
print('\n`) 
```

added in the code (see my comment below).

I prefer, it's more clear especially when there is an error message on TLS or `DNS entry not exist`.

But it's as you wish. Not very important.